src/components/view/Navigation/index.js

```diff
@@ -58,23 +44,40 @@ class Navigation extends React.Component {
         this.props.history.push(tabEl.props['data-route']);
     };
 
+    getMenuItemHandler (routeName)
+    {
+        var that = this;
+        return function ()
+        { 
+            that.setState({open: false});
+            that.props.history.push(routeName);
+        };    
+    }
+
+    handleLogout = () => {this.setState({open: false}); this.props.logout()}
+
     render () {
         const { user, logout } = this.props;
-        
-        const iconMenu = this.buildIconMenu({ logout });
+
+        // TODO: restore this back in once google authentication issue is resolved for me.
+        // if the user hasn't logged in, then render nothing.
+        // if (!user.uid || 0 === user.uid.length) return (<div/>);
 
         return (
             <div>
-                
-                {<AppBar
-                    title={'Welcome, ' + user.displayName}
-                    iconElementRight={iconMenu}
-                >
-                </AppBar>}
-
+                <IconButton onClick={this.handleToggle} style={{width: 60, height: 60, padding: 0}}>
+                    <Avatar size={60} icon={<PersonOutlineIcon/>}/>
+                </IconButton>
+                {'Welcome, ' + user.displayName}
+                <Drawer docked={false} width={200} open={this.state.open} onRequestChange={(open) => this.setState({open})}>
+                    <MenuItem onClick={this.getMenuItemHandler("contact")} primaryText="Contact" leftIcon={<PersonIcon/>}/>
+                    <MenuItem onClick={this.getMenuItemHandler("reports")} primaryText="Report" leftIcon={<EventNoteIcon/>}/>
+                    <MenuItem onClick={this.getMenuItemHandler("users")} primaryText="Add Users" leftIcon={<GroupAddIcon/>}/>
```

Comment:
Let's comment this line out for now... Great to have it ready to go for when we need it!